### DISCUSSION TITLE
Can't get TailwindCSS emmet abbreviation to work

### DISCUSSION THREAD

**Discussion by zozok3rr** [tailwindlabs/tailwindcss#6157] [Help]
I am sorry for the trouble. I just have to be more careful.

I don't know what I was doing, and I do believe I have to be more careful and precise.

**Comment by aggreggator** [comment]
Try to press TAB instead ENTER.

**Reply by zozok3rr** [reply]
Same result. It just finishes the line with selected text and that's about it.
What I have noticed, is that where is the regular expressions - it shows that it is Emmet in the selection, however on the Tailwindcss classnames it doesn't apply, but instead writes CSS properties

![image](https://user-images.githubusercontent.com/69513287/142770120-5c314b40-358a-4f46-b528-845ef21fa244.png)

**Reply by BorisAnthony** [reply]
I have this same issue.
I type, for example
`div`
i get the menu with just "emmet abbreviation"
`div.text-`
I get a fuller menu, first "div.text-" & "emmet abbreviation", and a full list of TailwindCSS options. The moment I select any of those (down key, or mouse) Emmet breaks and will not expand the `div.whatever` to HTML.

Maddening. 😅

**Reply by highjeans** [reply]
I currently can't get completions when writing in emmet syntax. @BorisAnthony Did you set any other settings other than "tailwindCSS.emmetCompletions" to true?
